soc/wch/ch32v/qingke_v4/vector.S

```diff
@@ -18,12 +18,15 @@ GTEXT(__initialize)
 
 SECTION_FUNC(vectors, ivt)
 	.option norvc
+	/* Jump to 0x08000008, into the main flash zone where j __start is */
+	lui	x5, 0x8000
+	jr	0x8(x5)
 	j       __start
 	.rept   CONFIG_VECTOR_TABLE_SIZE
 	.word	_isr_wrapper
 	.endr
 
 SECTION_FUNC(vectors, __start)
-	li 	a0, 3
+	li 	a0, 0xf
```

Comment:
At first, I wanted to ask what this change is about but I now have some idea:
You need this because lui and jr from above are now at 0x0 and then the table starts at 0x8, right?
I think this warrants a comment in code for the next person that tries to understand this line.

Reply:
oh, my bad.
But then vector table is one entry to small (check VECTOR_TABLE_SIZE and Table 9-2 in RM).
Could also verify using DMA2_CH11 interrupt

Reply:
You are right, the number should be 104, I'll make a PR for fixing this later.

Reply:
I have applied it there since I had to spend another hour updating this PR **again** since other things were merged before it was.

Reply:
> I have applied it there since I had to spend another hour updating this PR **again** since other things were merged before it was.

Feel free to mark other PRs as DNM if there are dependencies or preferences w.r.t merge order

Reply:
> > I have applied it there since I had to spend another hour updating this PR **again** since other things were merged before it was.
> 
> Feel free to mark other PRs as DNM if there are dependencies or preferences w.r.t merge order

Not only do i expected people to be pissed if I do that, but I also expect y'all take the normal time to merge that PR, eg 2 to 3 months counted in buisness days.

Reply:
> Not only do i expect people to be pissed if I do that, but I also expected y'all take the normal time to merge that PR, eg 2 to 3 months counted in buisness days so i could have the time to actually take a look at it.

I don't know what this is supposed to mean. PRs get merged when they're... mergeable?

Reply:
What it means is being unpredictable is worse than being slow, you cant just have trivial PRs taking months to merge because no one bothers taking a look or assigned person disappeared, while having big PRs being reviewed very partially (100 % certain nordicjm's review was only relevant to the board part, dont know about yours but i'm also sure there wasnt a overall architectural review because it didn't match the *current* architecture of the soc folders that **my PRs had to match being merged**) and merged near-instantly.

Reply:
like ffs we still cant use west flash on ch32v208 because we have to make sure <random person 458485> is ok with *fixing a obvious bug* but apparently breaking other PRs and making 4 contributors wait for months on end because no one can be bothered to actually review something agreed upon by 3/4th of the contributors of a platform.

Reply:
I want to apologize to you two, it's really not your fault, it's only the normal, insanely frustrating (especially in areas of low mainstream interest), process...